NEWS.md

```diff
@@ -1,3 +1,7 @@
+# vimpact 0.1.6
+
+* deal with multi-cohort routine for pulling coverage fvps. For details, see youtrack: https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-8760/fix-vimpact-for-routines-with-multiple-cohorts
```

Comment:
If external non-VIMC readers of the changelog can't access this document it's not worth including here. If access to external readers is important the YT doc contents should go somewhere in package docs.